src/commands/sites/create.js

```diff
@@ -95,6 +108,12 @@ SitesCreateCommand.flags = {
   'account-slug': flags.string({
     char: 'a',
     description: 'account slug to create the site under'
+  }),
+  'manual': flags.boolean({
```

Comment:
First pass here, I'm thinking we rename this to `--ci` or something, and that lets you set up CI for the new site, independent of the filesystem / current project you are in.  That would also mean we would need to acomidate the github configurator.  I can take it away from here if you want.